Identity/BusinessLayer/AutoMapperProfiles/Resolvers/UserOnlineResolver.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoMapper;
+using DataAccessLayer.Contracts;
+using DataAccessLayer.Entities;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+
+
+namespace BusinessLayer.AutoMapperProfiles.Resolvers
+{
+    public class UserOnlineResolver : IValueResolver<User, object, bool>
+    {
+        private readonly IUserRequestRepository _userRequestRepository;
+
+        public UserOnlineResolver(IUserRequestRepository userRequestRepository)
+        {
+            _userRequestRepository = userRequestRepository;
+        }
+
+        public bool Resolve(
+            User source,
+            object destination,
+            bool destMember,
```

Comment:
Why do you need these parameters here?

Reply:
Because they are input parameters of method IValueResolver.Resolve(), that was implemented